platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -382,6 +382,27 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)mapView:(MGLMapView *)mapView didDeselectAnnotation:(id <MGLAnnotation>)annotation;
 
 /**
+ Tells the delegate that one of its shape annotations was selected.
+ 
+ You can use this method to track changes in the selection state of annotations.
+ 
+ @param mapView The map view containing the annotation.
+ @param shapeAnnotation The shape annotation that was selected.
+ */
+- (void)mapView:(MGLMapView *)mapView didSelectShapeAnnotation:(MGLShape *)shapeAnnotation;
```

Comment:
These new APIs don't provide a shape "annotation". They do provide an `MGLShape`. Since they are intended for MGLPolyline(gon)(Feature) instances, should we remove the word "annotation" from the API and use a more specific type?

Reply:
MGLShape conforms to MGLAnnotation.